src/diffpy/structure/atom.py

```diff
@@ -202,11 +264,19 @@ def __copy__(self, target=None):
 
     @property
     def xyz_cartn(self):
-        """numpy.ndarray : Absolute Cartesian coordinates at `xyz`.
+        """
+        Obtain the absolute Cartesian coordinates at `xyz`.
 
-        Setting ``xyz_cartn`` or any of its items such as ``xyz_cartn[0]``
-        updates the fractional coordinates `xyz`.
+        Parameters
+        ----------
+        None.
+
+        Returns
+        -------
+        rv : numpy.ndarray
+            Absolute Cartesian coordinates at `xyz`.
```

Comment:
This is a property setter function.  Please convert to the format as in property example in https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy.